docker_setup_scripts/centos.sh

```diff
@@ -188,11 +193,38 @@ install_packages() {
 
 install_golang() {
   start_group "Installing Golang"
-  if [[ $centos_major_version -eq 7 ]]; then
-    rpm --import https://mirror.go-repo.io/centos/RPM-GPG-KEY-GO-REPO
-    curl -s https://mirror.go-repo.io/centos/go-repo.repo | tee /etc/yum.repos.d/go-repo.repo
+  if [[ $( uname -m ) == "aarch64" && $centos_major_version == 7 ]]; then
+    local go_version=1.19.2
+    local go_archive_name=go${go_version}.linux-arm64.tar.gz
+    local go_url=https://go.dev/dl/${go_archive_name}
+    local expected_sha256=b62a8d9654436c67c14a0c91e931d50440541f09eb991a987536cb982903126d
+    local tmp_dir=/tmp/go_installation
+    mkdir -p "${tmp_dir}"
+    (
+      cd "${tmp_dir}"
+      wget "${go_url}"
+      actual_sha256=$( sha256sum "${go_archive_name}" | awk '{print $1}' )
+      if [[ ${actual_sha256} != "${expected_sha256}" ]]; then
+        echo >&2 "Invalid SHA256 sum of ${go_archive_name}: expected ${expected_sha256}, got" \
+                 "${actual_sha256}"
+        exit 1
+      fi
+      tar xzf "${go_archive_name}"
```

Comment:
Missing a `-`?

Reply:
This works fine, this is how I always invoke tar.